docs/blog/author.yaml

```
@@ -392,3 +392,7 @@
   bio: Teacher, speaker, and community organizer who learned to code via Neopets. Loves theater, karaoke, and dogs.
   avatar: avatars/aisha-blake.jpg
   twitter: "@AishaBlake"
+- id: Rich Manalang
+  bio: Principal developer advocate at LaunchDarkly
+  avatar: avatars/rich-manalang.jpg
```

Looks like dangerbot is upset because of the file path. I believe it needs to be this.
```suggestion
  avatar: avatars/rich-manalang.jpeg
```